src/lib/Preview.js

```diff
@@ -415,13 +428,25 @@ class Preview extends EventEmitter {
      */
     download() {
         const { apiHost, queryParams } = this.options;
+        let downloadUrl;
 
         if (checkPermission(this.file, PERMISSION_DOWNLOAD)) {
             // Append optional query params
-            const downloadUrl = appendQueryParams(getDownloadURL(this.file.id, apiHost), queryParams);
+            downloadUrl = appendQueryParams(getDownloadURL(this.file.id, apiHost), queryParams);
             get(downloadUrl, this.getRequestHeaders()).then((data) => {
                 openUrlInsideIframe(data.download_url);
             });
+        } else if (canDownloadWatermarkedRep(this.file)) {
+            // If we can download the watermarked rep but not the original, we assume that the viewer is using the
+            // watermarked rep for preview, so we will download it.
+            // This allows the content URL to be downloaded by the browser
+            const downloadQueryParams = { response_content_disposition_type: 'attachment' };
+            const template = this.viewer.options.representation.content.url_template;
+            downloadUrl = appendQueryParams(
+                this.viewer.createContentUrlWithAuthParams(template, this.viewer.options.viewer.ASSET),
```

Comment:
Not ideal to access the viewer from Preview like this, thinking about alternatives.

Reply:
I'm thinking we should instead expose something like `this.viewer.getRepUrl()` that abstracts away some of this complexity and adds some error/null checks.

Then you could have:
```
const watermarkedRepUrl = this.viewer.getRepUrl();
const downloadUrl = appendQueryParams(this.viewer.appendAuthParams(watermarkedRepUrl), {
    response_content_disposition_type: 'attachment' // Tells browser to download, not display
});
openUrlInsideIframe(downloadUrl);
```

A few stylistic things:
- No need to have downloadQueryParams on a separate line since there aren't that many query param pairs and you don't need to optionally add any based on if statements
- Put `const downloadUrl` inside each block to save the line `let downloadUrl` above since you're not using `downloadUrl` outside of each block